upd_script/rc.sh

```diff
@@ -27,6 +27,10 @@ if [ -f /boot/hostnames ]
 then
   sudo mv /boot/hostnames /boot/hostname  
 fi
+# To change the Wifi access point name if hostname is changed using GUI
+if [[ -f /etc/hostapd/hostapd.conf ]] ; then
```

Comment:
Just wondering if there's an easy way to make this change only when it's necessary. For example, the test should be :
1. if the file exists (the current if statement)
2. it does not contain the current hostname
The goal is to avoid writing to the SD card if not necessary. Each write access has a *tiny* risk associated to it of corrupting the file or the whole card.